hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/operations/AbstractCustomCreateOperation.java

```diff
@@ -149,25 +149,19 @@ private void spawnChildMessage(@NonNull final MessageFrame frame) {
 
         final var childGasStipend = gasCalculator().gasAvailableForChildCreate(frame.getRemainingGas());
         frame.decrementRemainingGas(childGasStipend);
-        final var childFrame = MessageFrame.builder()
+        // Child frame automatically added to parent messageFrameStack
+        MessageFrame.builder()
                 .parentMessageFrame(frame)
                 .type(MessageFrame.Type.CONTRACT_CREATION)
-                .worldUpdater(frame.getWorldUpdater().updater())
```

Comment:
(This is the catastrophic "superfluous `.updater()` call" mixed in here with these other things being removed - had to look a couple of times to spot it.)